bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/internal/VoiceManagerImpl.java

```diff
@@ -151,7 +164,13 @@ protected void activate(BundleContext bundleContext, Map<String, Object> config)
 
     @Deactivate
     protected void deactivate() {
-        stopAllDialogs();
+        dialogProcessors.values().forEach(DialogProcessor::stop);
+        dialogProcessors.clear();
+        var dialogRegistrationFuture = this.dialogRegistrationFuture;
```

Comment:
I'm really sorry, I didn't notice before and have another request. We avoid using `var`. Since this is only cosmetic, it should be a fast-fix. I'll merge afterwards, promised.